runtime/fastly/host-api/host_api.cpp

```diff
@@ -944,6 +944,20 @@ Result<HostBytes> HttpReq::downstream_client_ip_addr() {
   return res;
 }
 
+Result<HostBytes> HttpReq::downstream_server_ip_addr() {
+  Result<HostBytes> res;
+
+  fastly_world_list_u8_t octets;
+  fastly_compute_at_edge_types_error_t err;
+  if (!fastly_compute_at_edge_http_req_downstream_server_ip_addr(&octets, &err)) {
+    res.emplace_err(err);
+  } else {
+    res.emplace(make_host_bytes(octets));
+  }
```

Comment:
Are we sure it's never an optional none?

Reply:
Yes we always have an IP for the servers in our pops which receive client requests 👍